InformationScripting/src/graph/PropertyMap.cpp

```diff
@@ -41,4 +47,19 @@ Property InformationScripting::PropertyMap::operator[](const QString& key) const
 	Q_ASSERT(false);
 }
 
+Property& InformationScripting::PropertyMap::operator[](const QString& key)
+{
+	for (auto& content : properties_)
+		if (content.first == key) return content.second;
+	qDebug() << "No object with name" << key;
```

Comment:
Here and above in the const version, is this what we want? Typically `[]` inserts a default element if the key is not found. Would it be bad to code it like usual?